impl/src/main/java/org/ehcache/impl/internal/spi/serialization/DefaultSerializationProvider.java

```diff
@@ -138,8 +184,14 @@ public void releaseSerializer(final Serializer<?> serializer) throws IOException
 
   @Override
   public void start(ServiceProvider<Service> serviceProvider) {
-    transientProvider.start(serviceProvider);
-    persistentProvider.start(serviceProvider);
+    addDefaultSerializerIfNoneRegistered(serializers, Serializable.class, (Class) CompactJavaSerializer.class);
```

Comment:
Needs a trick similar to what's done [here](https://github.com/ehcache/ehcache3/pull/1431/commits/cef191af7c1f4614244cb71900066117e68aac0e)